src/logging/file.lua

```diff
@@ -6,26 +6,32 @@
 -- @copyright 2004-2013 Kepler Project
 --
 -------------------------------------------------------------------------------
-
 local logging = require"logging"
+logging_fileHandler = {} -- store the file handlers
```

Comment:
this means the filehandles are kept alive forever. The filehandle should be an upvalue to the closure that is passed to `logging.new()` on line 43